easybuild/easyconfigs/m/magma/magma-2.5.4-fosscuda-2020a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "CMakeMake"
```

Comment:
Nit: Why the change to double quotes here after everything else is already single quote?

Reply:
I just rebased them on your updated 2019b version and it just followed from that. I have no strong opinions. I can change it if other care?

Reply:
Double quotes for the easyblock is consistent with everyother magma easyconfig:
```
$ grep easyblock *
magma-2.3.0-foss-2018a-CUDA-9.1.85.eb:easyblock = "ConfigureMake"
magma-2.3.0-fosscuda-2017b.eb:easyblock = "ConfigureMake"
magma-2.3.0-intelcuda-2017b.eb:easyblock = "ConfigureMake"
magma-2.4.0-fosscuda-2018b.eb:easyblock = "ConfigureMake"
magma-2.5.0-fosscuda-2018b.eb:easyblock = "ConfigureMake"
magma-2.5.1-fosscuda-2019a.eb:easyblock = "CMakeMake"
magma-2.5.1-fosscuda-2019b.eb:easyblock = "CMakeMake"
magma-2.5.4-fosscuda-2019a.eb:easyblock = "CMakeMake"
magma-2.5.4-fosscuda-2019b.eb:easyblock = "CMakeMake"
```

I do not mind either way.

Reply:
Then I must have overlooked that. If you don't mind I'd say change it so it is consistent  and new ECs based on this (this beeing the most recent ones) will have that too. No retest required anyway

Reply:
FWIW: I'm in favor of single quotes for string values that don't have spaces, but unless we make the style check test trip over that, it's pointless to discuss aspects like this.

Sticking to double quotes makes sense, it allows easier review with [`--review-pr`](https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#reviewing-easyconfig-pull-requests-review-pr)